### PR TITLE
convert byte strings in python3 to fix #145

### LINKED ISSUE #145: redraw commands in python3 are buffers not strings
This is a follow-up of #144. What happened is that in python3 the redraw instructions from nvim session are returned as byte buffers and not strings, and all the parsing expects strings. Besides that the GUI seems to work.

Is it supposed to work with python3? is anyone looking to fix this? I can take a look at it...

### PR DESCRIPTION
**Pull request by llpamies**
This is a simple fix to make the CLI work with python3. I took the code from one of the tests. It fixes #145.

### CODE REVIEW

File: neovim/ui/cli.py [hunk 1]
```diff
@@ -58,6 +60,11 @@ def main(ctx, prog, notify, listen, connect, profile):
         nvim_argv = shlex.split(prog or 'nvim --embed') + ctx.args
         nvim = attach('child', argv=nvim_argv)
 
+    if IS_PYTHON3:
+        # For Python3 we decode binary strings as Unicode for compatibility
+        # with Python2
```
Comment (bfredl) [inline neovim/ui/cli.py]: doesn't really make sense, we are doing this for compability with python3 :)

**Comment by bfredl** [comment]
Sorry for late reply. Tested this now and it works fine, so merged, execpt that I removed one comment as wasn't really correct and shouldn't be needed anyway. Thanks!

**Comment by justinmk** [comment]
@bfredl @fwalch @tarruda Was this published to pypi?

**Comment by bfredl** [comment]
@justinmk I'm tagging a release.

**Comment by bfredl** [comment]
I tagged but @tarruda needs to push to PyPI.

**Pull request closed without merging**